src/ml_board/frontend/Dashboard/src/redux/table/tableSlice.ts

```diff
@@ -70,6 +72,7 @@ export const { upsertOneRow, upsertManyRows } = tableSlice.actions;
 // create a set of memoized selectors
 export const {
     selectAll: selectAllRows,
+    selectById: selectRowById
```

Comment:
@vijulshah I added this for the experiment page #151
feel free to change `selectById` to `selectExperimentById` if you want 🤷
and you can simply use it as such (assuming that you have `expID`):
`const row = useAppSelector(state => selectRowById(state,expID));`
or
`const experiment = useAppSelector(state => selectExperimentById(state,expID));`